src/store/Network.ts

```diff
@@ -538,6 +561,27 @@ export default {
             const peerNodes: NodeInfo[] = await nodeRepository.getNodePeers().toPromise();
             commit('peerNodes', _.uniqBy(peerNodes, 'host'));
         },
+        async SET_TRANSACTION_DEADLINE({ commit, getters }, deadlineInHours = 2) {
+            const repositoryFactory: RepositoryFactory = getters['repositoryFactory'] as RepositoryFactory;
+            const deadline = await (await DeadlineService.create(repositoryFactory)).createDeadlineUsingServerTime(deadlineInHours);
+            switch (deadlineInHours) {
+                case 2:
```

Comment:
I would't put a dynamic value like a deadline in the store. The store is more for "infrastructure" objects, not to pass these kind of values. In general I don't like vuex, a lot harder to maintain and not type safe + adding dependencies to unrelated code

Reply:
in order to maintain the different deadlines in case of I aggregates we had to keep them somewhere to make sure they're consistent.